certif/app/components/enrolled-candidates.hbs

```diff
@@ -1,7 +1,7 @@
 <div class="panel">
   <div class="panel-header">
     <div class="panel-header__title">
-      Liste des candidats ({{this.args.certificationCandidates.length}})
+      Liste des candidats ({{@certificationCandidates.length}})
```

Comment:
👏 👏 👏 
Même en dehors d'eslint, je trouve ça tellement plus pratique / parlant le @ plutôt que this.args ...